openrtb_ext/bidders.go

```diff
@@ -103,6 +103,7 @@ const (
 	BidderYieldmo          BidderName = "yieldmo"
 	BidderYieldone         BidderName = "yieldone"
 	BidderZeroClickFraud   BidderName = "zeroclickfraud"
+	BidderBetween          BidderName = "between"
```

Comment:
Can we keep this list alphabetically ordered? Can we move entry below line 47?

Reply:
done, pushing soon.